index.js

```diff
@@ -0,0 +1,41 @@
+const assert = require('assert');
+const MongoDBAdapter = require('moleculer-db-adapter-mongo');
+
+module.exports = {
+  createLivenessChecker
+};
+
+/**
+ * Creates a Healthcheck checker for Mongo liveness
+ * This must be passed as a parameter to the HealthMiddleware liveness.createChecker
+ * @param {Object} broker to check liveness
+ */
+function createLivenessChecker(broker) {
+  return (next) => {
+    let error;
+    const servicesWithAdapter = broker.services.filter((service) => hasMongoAdapter(service));
```

Comment:
No need to create a lambda, pass directly the `hasMongoAdapter` function:

```js
broker.services.filter(hasMongoAdapter);
```

Reply:
Fixed in f0e66af7c5b51cc4003b855362d97b84de9c8e64